changelog/114.feature.rst

```diff
@@ -0,0 +1,12 @@
+Addition of the ``segment`` module and contains functions to segment granulation in images of the solar photosphere.
+The key functionality is contained in the `.segment` function, which takes a Map 
+and a thresholding method then performs the segmentation, with the segmented Map as output.
+Calls to segment look like:
+.. code-block:: python
+    segmented_data = segment(smap, resolution, skimage_method)
+where 
+*``smap`` is the Map containing an optical image of the solar photosphere.
+*``resolution`` is the spatial resolution of the telescope.
+*``skimage_method`` is the base scikit image segmentation technique to use.
+*``mark_dim_centers`` is a boolean flag indicating whether the user would like to mark dim intergranule centers for future exploration.
+We segment into tri-valued images with 0 = intergranule, 1 = granule, 1.5 = brightpoints, and if marking is desired, dim granule middles marked as 0.5.
```

Comment:
```suggestion
Addition of the ``segment`` module and contains functions to segment granulation in images of the solar photosphere.

The key functionality is contained in the `.segment` function, which takes a Map 
and a thresholding method then performs the segmentation, with a segmented Map as the output.

Calls to segment look like:

.. code-block:: python
    segmented_data = segment(smap, resolution, skimage_method)

where 

* ``smap`` is the Map containing an optical image of the solar photosphere.
* ``resolution`` is the spatial resolution of the telescope.
* ``skimage_method`` is the base scikit image segmentation technique to use.
* ``mark_dim_centers`` is a boolean flag indicating whether the user would like to mark dim intergranule centers for future exploration.

We segment into tri-valued images with 0 = intergranule, 1 = granule, 1.5 = bright points, and if marking is desired, dim granule middles marked as 0.5.
```

Reply:
Sorry! Missed the spaces when I copied in your suggestion.